src/WebUI/Pods/PodLog.tsx

```diff
@@ -54,6 +57,7 @@ export class PodLog extends BaseComponent<IPodLogProps, IPodLogState> {
                 uid: Util_String.newGuid(), // required to refresh the content
                 logContent: logContent || ""
             });
+            KubeFactory.telemetryService.scenarioEnd(Scenarios.PodLogs);

```

Comment:
if service not available or error you will not be able to get the end of the scenario. so you should check for service and then end the scenario, and have a final int he then/catch pattern and have the scenario end.